demos/CMakeLists.txt

```diff
@@ -109,8 +109,7 @@ add_demo(spectre_v1_btb_ca SYSTEMS Linux)
 # Ret2Spec -- speculative execution using return stack buffers creating a
 # call-ret disparity by inline assembly
 add_demo(ret2spec_callret_disparity
-         SYSTEMS Linux Darwin
-         PROCESSORS i386 x86_64 aarch64)
```

Comment:
Why did you have to remove this line?

Reply:
Because now all architectures are supported which is the default.